code/modules/clothing/spacesuits/spacesuits.dm

```diff
@@ -78,9 +81,11 @@
 	min_cold_protection_temperature = SPACE_SUIT_MIN_COLD_PROTECTION_TEMPERATURE
 	siemens_coefficient = 0.9
 	species_restricted = list("exclude")
-
+	equip_delay = 12 SECONDS
```

Comment:
That is an extremely long time.

Reply:
if you misclick just move to cancel it. the time applies to taking off as well as putting on

This particular time is balanced against the time a hardsuit takes to seal up